### ISSUE TITLE
Formatting issue with DataFrame's long time stamps

### ISSUE THREAD

**Issue by michaelaye** [holoviz/holoviz#246]
#### ALL software version info
(this library, plus any other relevant software, e.g. bokeh, python, notebook, OS, browser, etc)
panel: 0.9.3
holoviews: 1.13.1
pandas: 1.0.3
Browser: Chrome 80.0 on macOS 10.14.6

#### Description of expected behavior and the observed behavior

Following the Holoviz tutorial, I saw nicely right-hand formatted long time strings, but that doesn't happen here.
This has obviously something to do with pandas new timezone management, but I haven't learned yet how to influence the display of times properly since that change.

#### Complete, minimal, self-contained example code that reproduces the issue

```
import dask.dataframe as dd
import panel as pn
df = dd.read_parq

uet('../data/earthquakes.parq', columns=['time', 'place', 'mag']).reset_index(drop=True).persist()
pn.pane.DataFrame(df.head())
```
#### Screenshots or screencasts of the bug in action
<img width="425" alt="Screen Shot 2020-03-26 at 11 56 51" src="https://user-images.githubusercontent.com/69774/77679959-e3ed3400-6f58-11ea-8d26-1618552558f5.png">

while in the tutorial things looked like this:

<img width="350" alt="Screenshot 2020-03-26 11 58 39" src="https://user-images.githubusercontent.com/69774/77680174-33336480-6f59-11ea-8a31-9869e96d6618.png">

**Comment by michaelaye** [comment]
Work-around: 

Add this line after parsing the dataframe:

```python
import dask.dataframe as dd

df = dd.read_parquet('../../data/earthquakes.parq')

# new line:
df['time'] = df.time.dt.tz_convert(None).dt.floor('s')

df = df[~df.mag.isna()].persist()
```

**Comment by jbednar** [comment]
Fixed in #264. Thanks for the report!
